readthedocs/doc_builder/environments.py

```diff
@@ -910,7 +911,8 @@ def get_container_name(self):
                 project_name=self.project.slug,
             )
         else:
-            name = f'sync-project-{self.project.pk}-{self.project.slug}'
+            uuid = uuid.uuid4()
```

Comment:
You can take the first 8 or similar. Otherwise, we will probably loose the rest of the data (project pk and slug) when shrinked.